great_expectations/util.py

```diff
@@ -1899,6 +1905,37 @@ def get_context(  # noqa: PLR0913
     """
     project_config = _prepare_project_config(project_config)
 
+    if mode is not None:
+        if mode == "ephemeral":
+            return _get_ephemeral_context(
```

Comment:
All these `_get` methods have nullable returns - we need to ensure we return that type if the mode is requested